libs/ledger-live-common/src/api/Ledger.ts

```diff
@@ -12,37 +11,21 @@ type LedgerExplorer = {
 export const findCurrencyExplorer = (
   currency: CryptoCurrency
 ): LedgerExplorer | null | undefined => {
-  const config = getExplorerConfig()[currency.id];
-  if (!config) return;
-  const { id } = config;
   if (getEnv("SATSTACK") && currency.id === "bitcoin") {
     return {
       endpoint: getEnv("EXPLORER_SATSTACK"),
       id: "btc",
       version: "v3",
     };
   }
-  if (config.experimental && getEnv("EXPERIMENTAL_EXPLORERS")) {
-    const base = config.experimental.base;
-    const version = config.experimental.version;
-    return {
-      endpoint: getEnv(base),
-      id,
-      version,
-    };
-  }
 
-  const { base, version } = config.stable;
   return {
-    endpoint: getEnv(base),
-    id,
-    version,
+    endpoint: getEnv("EXPLORER"),
```

Comment:
Just a personal opinion but if not too painful could we give this a more explicit name like `EXPLORER_BASE_URL` ?

Reply:
Doing it correctly would imply changing every env variables in the env.ts file... kind of out of scope I think and it's just a env var